docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -59,49 +59,46 @@ If you have generated the application from the previous tutorial, you can find i
 
 [source,xml]
 ----
-<profiles>
-    <profile>
-        <id>native</id>
-        <build>
-            <plugins>
-                <plugin>
-                    <groupId>io.quarkus</groupId>
-                    <artifactId>quarkus-maven-plugin</artifactId>
-                    <version>${quarkus.version}</version>
-                    <executions>
-                        <execution>
-                            <goals>
-                                <goal>native-image</goal>
-                            </goals>
-                            <configuration>
-                                <enableHttpUrlHandler>true</enableHttpUrlHandler>
-                            </configuration>
-                        </execution>
-                    </executions>
-                </plugin>
-            </plugins>
-        </build>
-    </profile>
-</profiles>
+<profile>
+    <id>native</id>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-maven-plugin</artifactId>
+                <version>${quarkus.version}</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>native-image</goal>
+                        </goals>
+                        <configuration>
+                            <enableHttpUrlHandler>true</enableHttpUrlHandler>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+</profile>
 ----
 
 We use a profile because, you will see very soon, packaging the native image takes a _few_ seconds.
 
 Create a native executable using: `mvn package -Pnative`.
 
-In addition to the regular files, the build also produces `target/quarkus-quickstart-runner`.
-You can run it using: `./target/quarkus-quickstart-runner`.
+In addition to the regular files, the build also produces an executable: `target/getting-started-1.0-SNAPSHOT-runner`.
+You can run it using: `target/getting-started-1.0-SNAPSHOT-runner`.
```

Comment:
So the issue here is that the quickstart code has `<finalName>quarkus-quickstart</finalName>` so it's indeed correct.

We should probably drop that from the quickstart code (in master so that users have the fix immediately). @cescoffier WDYT?

Reply:
The `finalName` is used to have the same "filename" in the 3 first quickstarts as they are 
 part of the same series. 

However, I'm in favor of dropping the  `finalName` and explain in a comment that if they follow the instructions from the quickstart, the `artifactId` would be different.

WDYT?

Reply:
I don't think having the same filename is a must have. They will follow the instructions anyway so the filenames can be different IMHO.

Reply:
Then, let's drop the tag.